src/codeu/chat/client/simplegui/ChatGuiFX.java

```diff
@@ -62,10 +63,14 @@ public void run(String [] args) {
             System.exit(1);
         }
 
+    }*/
+    
+    public static void setContext(Controller controller, View view) {
+    	clientContext = new ClientContext(controller, view);
     }
 
     public void launch(Controller controller, View view) {
-        this.clientContext = new ClientContext(controller, view);
+    	setContext(controller, view);
```

Comment:
Static stuff should be accessed statically. So this should be `ChatGuiFX.setContext()` instead. It's not a compiler problem, but it makes it more obvious that something is static.